erinspace/osf.io
# [#PLAT-710]Add node logs when users turn on/off request access

## Purpose
Add logs for enabling or disabling node access request


## Changes



## QA Notes
1. Create a projrect
2. go to the project settings page
3. select the tab access requests and check the box to "Allow users to request access to this project."
4. go back to the project overview page and check the log
5. go to the project settings page again
6. select the tab access requests and uncheck the box to "Allow users to request access to this project."
7. go back to the project overview page and check the log
Log expected to see:
<img width="598" alt="screen shot 2018-03-15 at 3 16 21 pm" src="https://user-images.githubusercontent.com/4974056/37486120-e6aea496-2863-11e8-8b33-5a0c322f6895.png">



## Side Effects



## Ticket
https://openscience.atlassian.net/browse/PLAT-710
